icu4c/source/test/intltest/itrbnf.cpp

```diff
@@ -1604,6 +1605,38 @@ IntlTestRBNF::TestThaiSpellout()
     delete formatter;
 }
 
+void
+IntlTestRBNF::TestNorwegianSpellout()
+{
+    UErrorCode status = U_ZERO_ERROR;
+    RuleBasedNumberFormat* noFormatter
+        = new RuleBasedNumberFormat(URBNF_SPELLOUT, Locale("no"), status);
+    RuleBasedNumberFormat* nbFormatter
+        = new RuleBasedNumberFormat(URBNF_SPELLOUT, Locale("nb"), status);
+
+    if (U_FAILURE(status)) {
+        errcheckln(status, "FAIL: could not construct formatter - %s", u_errorName(status));
+    } else {
+        static const char* testDataDefault[][2] = {
+            { "1", "\\u00E9n" },
+            { "2", "to" },
+            { "3", "tre" },
+            { "4", "fire" },
+            { "101", "hundre og \\u00E9n" },
+            { "123", "hundre og tjue\\u00ADtre" },
+            { "1,001", "tusen og \\u00E9n" },
+            { "1,100", "tusen hundre" },
+            { "6.789", "seks komma sju \\u00E5tte ni" },
+            { "-5.678", "minus fem komma seks sju \\u00E5tte" },
+            { NULL, NULL }
+        };
+        doTest(noFormatter, testDataDefault, TRUE);
+        doTest(nbFormatter, testDataDefault, TRUE);
+    }
+    delete nbFormatter;
```

Comment:
ok, but LocalPointer would be nicer

Reply:
Yes, but I was following existing precedent in the file to make it easier to compare the various tests for different locales. Probably what we need is a ticket to make a global change (or at least changes for entire files at one time rather than piecemeal).